CMakeLists.txt

```diff
@@ -76,7 +91,7 @@ set( main_lib_dest wpilib/lib )
 set( java_lib_dest wpilib/java )
 set( jni_lib_dest wpilib/jni )
 
-if (MSVC OR FLAT_INSTALL_WPILIB)
+if (MSVC OR WITH_FLAT_INSTALL_WPILIB)
```

Comment:
If MSVC is always OR’d with flat install, should we just force the option ON with MSVC?